man/systemd-ask-password.xml

```diff
@@ -165,6 +165,14 @@
         usernames. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--emoji=yes|no|auto</option></term>
+
+        <listitem><para>Whether or not to show an emoji. The default is
```

Comment:
maybe say explicitly insert the emoji we show here, i.e. (`Controls whether or not to prefix the query with a lock and key emoji (🔐), if the TTY settings permit this.`)

At least the compiled HTML version of the docs should have no issue including an emoji, and I think the man page version should be fine too